### PR TITLE
Feature : Add Donate Screen to Home

### PR DESCRIPTION
**Pull request by smittal1209**
**Description of PR**
Allow the users to directly contribute to Government or NGOs Relief Fund.

**Type of PR**
- [ ] Bugfix
- [x] New feature

**Checklist**
- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

**Screenshots**

Desktop : 

Screenshot 2020-04-01 at 6 21 39 PM

Mobile :

Screenshot 2020-04-01 at 6 27 34 PM

### CODE REVIEW

**Comment by smittal1209** [comment]
@tuxdna @tecoholic @guneetnarula @sriksm19 Can anyone please check what is the issue with Zeit Deployment?

File: src/components/expansion.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableRow from '@material-ui/core/TableRow';
+
+const ExpansionPanelItem = (props) => {
+  const rows = [];
+  const links = [];
+  props.props.content !== undefined &&
+    Object.entries(props.props.content).map(([key, value]) => {
```
Comment (sriksm19) [inline src/components/expansion.js]:
```suggestion
    Object.entries(props.props.content).forEach(([key, value]) => {
```

File: src/components/expansion.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      rows.push({key, value});
+    });
+  props.props.links !== undefined &&
+    Object.entries(props.props.links).map(([key, value]) => {
```
Comment (sriksm19) [inline src/components/expansion.js]:
```suggestion
    Object.entries(props.props.links).forEach(([key, value]) => {
```

**Comment by sriksm19** [comment]
where do i see what check has failed? i suppose i dont have access

**Comment by tuxdna** [comment]
https://zeit.co/covid19india/covid19india-react/ctcz07yha

But apparently that page is now missing.

@smittal1209 did you try deploying it on a local instance on your machine ?

**Comment by tuxdna** [comment]
@smittal1209 you can try making and another commit to this same PR and that will trigger a new deployment on ZIET. This time keep an eye on what failed at ZIET side.

**Comment by smittal1209** [comment]
@sriksm19 Thanks for the quick update and suggestions. Your suggestions has been resolved and also the Zeit deployment has been successfully done. 
Please review and accept this pull request. 
Moreover, the data for this feature will be given by the API which is also pending in the Pull Request [here](https://github.com/covid19india/api/pull/55).
If possible, please review and accept that as well so that I can start on working on making the data dynamic from your database.

**Comment by jeremyphilemon** [comment]
Hey thanks for the PR! As mentioned in the [contributing](https://github.com/covid19india/covid19india-react/blob/master/CONTRIBUTING.md) guide, please have an issue filed first, have it accepted, then assigned to you before submitting a PR :)

This will help us discuss any changes/requirements before work is done on it!

**Pull request closed without merging**